python/MeshData.sip

```diff
@@ -24,12 +24,12 @@ class MeshData
 public:
     MeshData();
     virtual ~MeshData();
-    PyObject* getFacesAsBytes();
-    PyObject* getVerticesAsBytes();
+    std::vector<char> getVerticesAsBytes();
```

Comment:
Absolutely!
I initially considered this, then wasn't sure if it was consistent with the rest of the source code. There was a typedef from unsigned int to uint somewhere, that confused me a bit.

The typedef seems a bit excessive to me, but as long as it's properly namespaced I guess there's no harm.